src/commands/ptal.ts

```diff
@@ -97,32 +113,39 @@ const generateReplyFromInteraction = async (description: string, github: string,
 	let content = "";
 	let pr_state: PullRequestState = 'PENDING';
 
-	let githubURL = TryParseURL(githubOption, interaction);
-	if(githubURL)
+	//github
 	{
-		const pathSections = githubURL.pathname.split("/");
-		pathSections.shift();
-		if(githubURL.hostname != "github.com" || pathSections.length != 4 || pathSections[2] != "pull" || !Number.parseInt(pathSections[3]))
+		const githubRE = /https:\/\/github\.com\/(?<ORGANISATION>[^\/]+)\/(?<REPOSITORY>[^\/]+)\/pull\/(?<NUMBER>\d+)/;
+		const otherRE = /((?<ORGANISATION>[^\/]+)\/)?(?<REPOSITORY>[^(#|\s)]+)(#|\s)(?<NUMBER>\d+)/;
```

Comment:
In the second regex, it may be worth to also match `/<org?>/<repo?>/pull/<number>` as well, because currently the options are either a full URL or `/<org?>/<repo?>#<number>` 
The latter doesn't actually resolve in github (except `#<number>` inline, but that only works for a single repo)

Reply:
Also `/org/repo/ 123` is valid because of `(#|\s)`, I don't think that was intentional but feel free to correct me if I'm wrong

![image](https://github.com/withastro/houston-discord/assets/55956895/64c83777-916e-4863-b8c8-a314a1c6f194)